### DISCUSSION TITLE
Figure is not centered with :class:

### DISCUSSION THREAD

**Discussion by PierreMarchand20** [pradyunsg/furo#460] [Potential Issue] [answered]
When using `only-light`, `figure` is not centered. For example:
```
.. figure:: ../_static/svg/git/versioning.drawio.svg
   :align: center
   :class: only-light
```

<img width="796" alt="image" src="https://user-images.githubusercontent.com/15056323/174981436-c4babaa7-60e2-4616-967b-fae3c70d7d42.png">

When removing `only-light`, it is centered.
```
.. figure:: ../_static/svg/git/versioning.drawio.svg
   :align: center
```

<img width="800" alt="image" src="https://user-images.githubusercontent.com/15056323/174981602-12d08347-2a6f-4234-8072-22b3570bd049.png">

Note that it only happens with `figure`, but not with `image`:
```
.. image:: ../_static/svg/git/versioning.drawio.svg
   :align: center
   :class: only-light
```

<img width="788" alt="image" src="https://user-images.githubusercontent.com/15056323/174982029-42aac4c3-b3f5-43d9-9c38-78262c80245c.png">

**Comment by fwitte** [comment, marked as answer]
I do have the same Issue, I put a case to reproduce inside this zip folder following the nice template from #103 to reproduce the issue. [furo-bug.zip](https://github.com/pradyunsg/furo/files/9353494/furo-bug.zip)

**Reply by fwitte** [reply]
Just found the solution, it is in #423 @PierreMarchand20, if you are still searching.

**Reply by PierreMarchand20** [reply]
Thanks! I did not see this thread, as they said, it is worth updating the doc ^^
